src/lib/proxy/create-context.test.ts

```diff
@@ -55,13 +55,23 @@ test('will not blow up if properties is an array', () => {
         properties: ['some'],
     });
 
-    // console.log(context);
-
     expect(context.userId).toBe('123');
     expect(context).not.toHaveProperty('tenantId');
     expect(context).not.toHaveProperty('region');
 });
 
+test('will respect environment set in context', () => {
+    const context = createContext({
+        userId: '123',
+        tenantId: 'some-tenant',
+        environment: 'development',
+        region: 'eu',
+        properties: ['some'],
+    });
+
+    expect(context.environment).toBe('development');
```

Comment:
I would like to see another test proving environment will not be `development` when not set on context.